src/solc_0.8/StarterPack/StarterPackV2.sol

```diff
@@ -358,25 +360,13 @@ contract StarterPackV2 is PurchaseValidator, ERC2771Handler {
         require(IERC20(_sand).transferFrom(buyer, paymentRecipient, amountForDestination), "PAYMENT_TRANSFER_FAILED");
     }
 
-    /// @dev this override is required
+    /// @dev this override is required; two or more base classes define function
     function _msgSender() internal view override(Context, ERC2771Handler) returns (address sender) {
-        if (isTrustedForwarder(msg.sender)) {
-            // The assembly code is more direct than the Solidity version using `abi.decode`.
-            // solhint-disable-next-line no-inline-assembly
-            assembly {
-                sender := shr(96, calldataload(sub(calldatasize(), 20)))
-            }
-        } else {
-            return msg.sender;
-        }
+        return ERC2771Handler._msgSender();
     }
 
-    /// @dev this override is required
+    /// @dev this override is required; two or more base classes define function
     function _msgData() internal view override(Context, ERC2771Handler) returns (bytes calldata) {
-        if (isTrustedForwarder(msg.sender)) {
-            return msg.data[:msg.data.length - 20];
-        } else {
-            return msg.data;
-        }
+        return ERC2771Handler._msgData();
```

Comment:
not sure why thoses changes are here, careful with the merge

Reply:
removed